zubhub_backend/zubhub/creators/utils.py

```diff
@@ -456,3 +458,90 @@ def send_notification(users: List[Creator], source: Creator, contexts,
 #         EmailAddress.objects.create(
 #             user=user, email=email, primary=False, verified=False
 #         )
+
+def set_badge_view_category(creator):
+    views_count= Project.objects.filter(creator= creator).aggregate(Sum(('views_count')))["views_count__sum"]
+
+    List = ["Getting Famous", "Person of Interest", "Popular Projects", "Idea Factory"]
+
+    creator.badges.remove(Badge.objects.get(badge_title = List[3]))
+    creator.badges.remove(Badge.objects.get(badge_title = List[2]))
+    creator.badges.remove(Badge.objects.get(badge_title = List[1]))
+    creator.badges.remove(Badge.objects.get(badge_title = List[0]))
+    
+    if(views_count > 100000):
+        creator.badges.add(Badge.objects.get(badge_title = List[3]))
+
+    elif(views_count > 5000):
+        creator.badges.add(Badge.objects.get(badge_title = List[2]))
+
+    elif(views_count > 100):
+        creator.badges.add(Badge.objects.get(badge_title = List[1]))
+
+    elif(views_count > 10):
+        creator.badges.add(Badge.objects.get(badge_title = List[0]))
+    
+def set_badge_like_category(creator):
+    likes_count= Project.objects.filter(creator= creator).aggregate(Sum(('likes_count')))["likes_count__sum"]
+    List= ["Interesting Projects", "Favourite Kid", "Captain Projects"]
+
+    creator.badges.remove(Badge.objects.get(badge_title = List[2]))
+    creator.badges.remove(Badge.objects.get(badge_title = List[1]))
+    creator.badges.remove(Badge.objects.get(badge_title = List[0]))
+
+    if(likes_count > 1000):
```

Comment:
same comment for counts check like above. or try to see if there is a better way to reduce if/else blocks with a single function/constant/configuration like constant which returns badge_title based on the count.
We can do this for all 4 type of count checks in this file.